src/components/mapComp/mapComponent.module.scss

```diff
@@ -0,0 +1,15 @@
+.googleMapContainer {
```

Comment:
Also if screen is smaller than 400px map is too wide. Change min-width or delete. I think width should be enough

Reply:
I know that, but I think if screen is small, it is more comfortable when displayed element take as much place as possible..